ramin/js/module-57/node_implement/isNil.js

```diff
@@ -0,0 +1,11 @@
+// return true if parameter is falsy, false otherwise
+function isNil(param) {
+    return !(param);
+}
+
+
+module.exports = {
+    isNil
```

Comment:
This is not a default export.

Reply:
hi i solve this problem in two way 
in this way we can't use default keyword because we return an object 
but in an other way we can use default keyword.
but advantage of this way that we don't want to define an external file (package.json)

Reply:
I know this method doesn't support `default` keyword, but you can write it somehow to simulate a default export.

Reply:
you mean i write like this ?
export default  function isNil(param) {
    return !(param);
}